src/Microsoft.TestPlatform.AdapterUtilities/Microsoft.TestPlatform.AdapterUtilities.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TestPlatformRoot Condition="$(TestPlatformRoot) == ''">..\..\</TestPlatformRoot>
+  </PropertyGroup>
+  <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
+  
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);uap10.0;</TargetFrameworks>
```

Comment:
Just an observation on something that is likely to come up in the future: Since managed test frameworks will probably require this reference to be added in their framework layer (as opposed to the VS / adapter layer) and since the framework layer will likely be consumed in older versions of VS (as well as outside of VS), test frameworks may end up needing to conditionally add package reference for `MS.TP.AdapterUtilities` to only a subset of their supported targets and / or conditionally opt in to the FQN functionality in their framework layer only in cases where they know that the caller is VS. Alternatively, if they can't consume the package they may need to maintain a parallel implementation of the FQN spec or copy the Reflection code from the current repo...